src/components/Trade/TradeInput.tsx

```diff
@@ -203,10 +221,15 @@ export const TradeInput = () => {
       quote,
       sellAssetBalanceCrypto,
     )
-    setValue('action', TradeAmountInputField.SELL_CRYPTO)
-    setValue('sellTradeAsset.amountCryptoPrecision', maxSendAmount)
-    setValue('amount', maxSendAmount)
-    setValue('isSendMax', true)
+    swapperDispatch({
+      type: SwapperActionType.SET_VALUES,
+      payload: {
+        sellTradeAsset: { ...sellTradeAsset, amountCryptoPrecision: maxSendAmount },
+        action: TradeAmountInputField.SELL_CRYPTO,
+        isSendMax: true,
+        amount: maxSendAmount,
```

Comment:
Also not introduced in this PR, but sounds like `isSendMax` and `amount` should be mutually exclusive and we should be able to get the max send amount somewhere else instead of passing it down in case we're in a sendMax context?